deploy/servctl_utils/deploy_command_utils.py

```diff
@@ -269,14 +269,14 @@ def deploy(deployment_target, components, output_dir=None, runtime_settings={}):
     if not components:
         raise ValueError("components list is empty")
 
-    if components:
-        run('deploy/kubectl_helpers/utils/check_context.sh {}'.format(deployment_target.replace('gcloud-', '')))
+    # if components:
```

Comment:
Were the changes in this file intentional for this PR?

Reply:
nope, reverted these back, good catch